ksql-engine/src/main/java/io/confluent/ksql/KsqlExecutionContext.java

```diff
@@ -88,6 +92,34 @@
    */
   PreparedStatement<?> prepare(ParsedStatement stmt);
 
+  /**
+   * Executes a query using the engine's primary service context.
```

Comment:
Shouldn't this comment indicate that this is executing it using a supplied service context? Also, should there be another method for the primary service context?
`  TransientQueryMetadata executeQuery(
      ConfiguredStatement<Query> statement
  );`

Reply:
> Shouldn't this comment indicate that this is executing it using a supplied service context? 
good catch

> should there be another method for the primary service context?
I don't want to add it until there's something to use it.

Reply:
Turns out there is a use case for it 😁